jet_tracking/gui/windows/mainWindow.py

```diff
@@ -1,34 +1,70 @@
 import logging
-
+import sys
 from context import Context
 from gui.views.jetImageView import JetImageView
 from gui.views.jetTrackerView import JetTrackerView
 from gui.windows.mainWindowUi import Ui_MainWindow
 from gui.windows.simulationWindow import SimWindow
 from PyQt5.Qt import Qt
-from PyQt5.QtCore import QCoreApplication
+from PyQt5.QtCore import QCoreApplication, QSize
 from PyQt5.QtWidgets import (QAction, QLabel, QMainWindow, QSizePolicy,
                              QTabWidget)
 from signals import Signals
+sys.path.append('/cds/group/pcds/epics-dev/ajshack/jet_tracking/jet_tracking/pyqt-stylesheets/')
```

Comment:
This vendoring a submodule via sys.path.append construct is an antipattern that makes it harder to predict the overall import structure of the application. It's something that we'll need to clean up after this PR. We'll want to continue to use these stylesheets but in a more controlled way e.g. either installed into the python env or at least importable without any in-python pathmunging.